src/base-model.ts

```diff
@@ -115,6 +120,18 @@ export default abstract class Model<T> {
     }
   }
 
+  private isValidItem(item: unknown): item is T {
+    try {
+      this.pkValue(item as T);
+      if (this.sk != null) {
```

Comment:
You should verify truthiness instead with `!!this.sk`. Because `this.sk != null` would return `true` with `this.sk = ''` :)